packages/ember/package.json

```diff
@@ -22,7 +22,7 @@
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint . --cache --cache-location '../../eslintcache/'",
-    "pack": "npm pack",
+    "package": "run-s prepack && npm pack && run-s postpack",
```

Comment:
Looking at this, it probably makes more sense to inline the `pre/postpack` scripts under `package` instead of having three scripts here... is there anything I'm missing that would break this?

Reply:
Nope, don't think so. Good idea.